store/src/java/com/zimbra/cs/ldap/unboundid/LdapConnUtil.java

```diff
@@ -44,6 +44,7 @@ static SocketFactory getSocketFactory(LdapConnType connType, boolean allowUntrus
     static LDAPConnectionOptions getConnectionOptions(LdapServerConfig ldapConfig) {
         LDAPConnectionOptions connOpts = new LDAPConnectionOptions();
 
+        connOpts.setAllowConcurrentSocketFactoryUse(true);
```

Comment:
would recommend adding a comment on what happens when we set it to true since this is a bug fix

something like this : 
`allowConcurrentSocketFactoryUse - Indicates whether to allow a socket factory instance to be used to create multiple sockets concurrently.
`

https://docs.ldap.com/ldap-sdk/docs/javadoc/com/unboundid/ldap/sdk/LDAPConnectionOptions.html#setAllowConcurrentSocketFactoryUse-boolean-